README.md

```diff
@@ -1,3 +1,2 @@
-# thesis
-# note on setup: you will need to enter the Pokemon-Showdown directory and run node build. (necessarily this requires installing node if you don't have it).
-# also, you will need to install xdotool.
+# Setup
+Change directory to Pokemon-Showdown. Run "node build". (If you haven't installed node yet, you will need to). Then cd .. back to the previous directory. To start learning, run "python pkmn.py". Note that env_pkmn.py, which pkmn.py depends on, uses a relative path to get to its dependences in Pokemon-Showdown. So if you run this from some other directory, it will be broken.
```

Comment:
Mention something here about copying save_stable.p to save.p, if this is one's first time.

Reply:
Done at https://github.com/3141592654/thesis-mason-koch/commit/fe198fc9d5b4ef3d753dd3f5d351c1bd312d0acd